test/parallel/test-abortsignal-drop-settled-signals.mjs

```diff
@@ -0,0 +1,60 @@
+// Flags: --expose_gc
+
+import * as common from '../common/index.mjs';
+
+if (common.isASan) {
+  common.skip('ASan messes with memory measurements');
+}
```

Comment:
I got this from another test that involves memory testing. Let me know if I should remove this.

Reply:
We're not actually measuring memory in this test so likely unnecessary?

Reply:
Makes sense. Will remove.